commit 86955bf62e5690f65995ee8055b1ed529f80842b
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Fri Aug 7 09:53:56 2015 +0200

    Fix sysdeps/i386/fpu/s_scalbn.S build
    
    * math/Versions (libc: GLIBC_2_22): New (empty) version set.

diff --git a/math/Versions b/math/Versions
index 513ab14..5ab9123 100644
--- a/math/Versions
+++ b/math/Versions
@@ -29,6 +29,8 @@ libc {
     # s*
     scalbln; scalblnf; scalblnl;
   }
+  GLIBC_2.22 {
+  }
 }
 
 libm {
